website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -235,6 +235,24 @@ Examples of how to use [AKS with Advanced Networking](https://docs.microsoft.com
 
 * `load_balancer_sku` - (Optional) Specifies the SKU of the Load Balancer used for this Kubernetes Cluster. Possible values are `basic` and `standard`. Defaults to `basic`.
 
+* `load_balancer_profile` - (Optional) A `load_balancer_profile` block. Dependent upon `load_balancer_sku` being `standard`.
```

Comment:
to match the other resources could we make this:

```suggestion
* `load_balancer_profile` - (Optional) A `load_balancer_profile` block. This can only be specified when `load_balancer_sku` is set to `standard`.
```